components/organisms/TopNav/top-nav.tsx

```diff
@@ -51,7 +51,7 @@ const Nav = ({ className, name = "Main" }: { className?: string; name?: string }
   const { onboarded } = useSession();
 
   const { data: gitHubUser } = useFetchUser(user?.user_metadata.user_name);
-  const userInterest = gitHubUser?.interests.split(",")[0] || "hacktoberfest";
+  const userInterest = gitHubUser?.interests.split(",")[0] || "javascript";
```

Comment:
I know this was the code from before, but interests might not be languages which is what we display aside from Hacktoberfest.

For another PR, but filtering a user's interests on the available list of interests and grab the first one, otherwise default to JavaScript.